test/functional/db.test.js

```diff
@@ -183,7 +183,7 @@ describe('Db', function() {
           });
         };
 
-        automatic_connect_client.once('close', closeListener);
+        client.once('close', closeListener);
```

Comment:
`automatic_connect_client` seems particularly poorly named

Reply:
Agreed, perhaps `auto_reconnect_client` would be better? There's also a mix of camel/snake case for variables being used in these tests, would be nice to make that more consistent eventually.

Reply:
Well I was actually thinking "this thing is a `Db` not a `MongoClient`!"

Reply:
Oh right, didn't read that one correctly. I'll clean that test up a bit.